index.html

```diff
@@ -56,17 +72,28 @@ <h2 class="section-header-subtitle">I’m a Software Developer</h2>
 
     <main>
       <section class="container-2">
+
+
         <div class="w-container">
           <h2 class="my-work">My Recent Works</h2>
+            <!-- <div class="hr-box"> -->
+              <img src="./images/second-divider.png" class="w-container-img" alt="divider">
           <hr class="hr">
+            <!-- </div> -->
         </div>
-        <div class="image-paragraph">
+
+        <!-- <div class="multi-story"> -->
+          <div class="image-paragraph">
           <img class="img-pd" src="./images/bg.png" alt="place holder">
           <div class="multiple-column">
            <h2 class="multi-column-h1">Multi-Post Stories</h2>
            <p class="multi-column-det">A daily selection of privately personalized reads; no accounts or sign-ups required.
             has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
            </p>
```

Comment:
- [ ] Please add the right font family and font-weight to all heading text and try to stick as much as possible to the given Figma design.
- [ ] Kindly fic the margin between the head text and the horizontal line.

| Figma Design  | Your Design |
| ------------- | ------------- |
|  ![image](https://user-images.githubusercontent.com/68796232/159834756-8b08035d-344a-400a-b71c-aeb399dcf62a.png) | ![image](https://user-images.githubusercontent.com/68796232/159834658-fba1bee4-c7d7-4f4c-b541-1690413c006b.png) |

Reply:
- [ ]  Added the font family and the font-weight to all the heading texts as provided in Figma. I mean the name of the font family and a fallback general name too.
You can confirm in the CSS with the class name 'my-work'.
![my](https://user-images.githubusercontent.com/86200356/159927953-40e17c0c-628f-4382-9cc4-dc055aab122b.jpg)

- [ ]  Added the font family and the font-weight to all the heading texts as provided in Figma. I mean the name of the font family and a fallback general name too.
You can confirm in the CSS with the class name 'multi-column-h1'.
![multi](https://user-images.githubusercontent.com/86200356/159928770-c5b9a814-2771-4d24-983b-6340c9ce633c.jpg)

- [ ] I fixed the margin between the header text and the line.
![margin](https://user-images.githubusercontent.com/86200356/159929235-2e78b3da-256d-407b-bc37-ff4b86317b13.jpg)